packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -203,13 +211,17 @@ export const PropertyTypeRow = ({
       <TableCell>
         <PropertyExpectedValues property={property} />
       </TableCell>
-      <TableCell sx={{ textAlign: "center" }}>
-        <Checkbox />
-      </TableCell>
-      <TableCell sx={{ textAlign: "center" }}>
-        <Checkbox />
-      </TableCell>
-      <TableCell sx={{ px: "0px !important" }}>
+
+      <MultipleValuesCell propertyIndex={propertyIndex} />
+
+      <CenteredTableCell sx={{ textAlign: "center" }}>
+        <Checkbox
+          checked={required}
```

Comment:
You shouldn't need to use `watch` for this. `register` works by passing a ref and manually updating the value when it changes to avoid component re-renders. You're breaking that performance optimisation here. Does `Checkbox` not take a ref or something? Have you looked into how to use react-hook-form with checkboxes?